website/src/pages/documents/document.page.tsx

```diff
@@ -347,7 +355,13 @@ export const DocumentTitleHeader = (p: {
           audioUrl={p.doc.audioRecording.resourceUrl}
           showProgress
         />
+        {p.doc.audioRecording && (
+        <div>
+        {!isMobile ? <a href={p.doc.audioRecording?.resourceUrl} download={"audio"}><Button>Download Audio</Button></a> : null}
```

Comment:
To avoid the CORS error, are there any links to the audio file that aren't the `resourceUrl`? Looks like you can set a `download` attribute on files that don't link to a different domain: https://stackoverflow.com/a/64719808